erpnext/regional/report/gstr_1/gstr_1.py

```diff
@@ -144,17 +144,8 @@ def get_invoice_items(self):
 		""" % (self.doctype, ', '.join(['%s']*len(self.invoices))), tuple(self.invoices), as_dict=1)
 
 		for d in items:
-			item_details = {}
-			item_details[d.item_code] = d.base_net_amount
-
-			if d.parent in self.invoice_items:
-				parent_dict = self.invoice_items[d.parent]
-				if d.item_code in parent_dict:
-					item_details[d.item_code] += parent_dict[d.item_code]
-				else:
-					item_details.update(parent_dict)
-
-			self.invoice_items[d.parent] = item_details
+			if not self.invoice_items or d.item_code not in self.invoice_items.values():
```

Comment:
I think `self.invoice_items.values()` is not correct.

It should be `self.invoice_items.get(d.parent, {})`

Reply:
True. Your requested changes is done.